### DISCUSSION TITLE
Custom item with the same meter

### DISCUSSION THREAD

**Discussion by DreamyNiri** [misthero/dnd5e-spellpoints#93] [Q&A]
Hello, let me start off with saying that I am not a code expert and merely a beginner.

I really love how the purple meter for spell points is displayed under the hit dice and really wanted to make the same meter for custom items with charges.

I would appreciate if i could be told what lines i could copy and edit to correctly set it up without tying it to spells and magic since all I want is just a fancy meter on specific custom items.

**Comment by misthero** [comment]
hello, glad you like it, I can try to point you in the right direction, the function injecting the meter is `alterCharacterSheet` line 821 on spellpoints.js file, that function is hooked to the "renderActorSheet5eCharacter2" hook and it uses the template spell-points-sheet-tracker.hbs you need to pass the correct variables to the hbs file, and change it when needed. The rest is in the css file.

anyway I don't think that just copying a few lines of codes would make it work for every item, the data is very specific, but maybe you can get the right idea.
